parser/CMakeLists.txt

```diff
@@ -95,6 +95,7 @@ add_library(quickstep_parser_ParseLiteralValue ParseLiteralValue.cpp ParseLitera
 add_library(quickstep_parser_ParseOrderBy ParseOrderBy.cpp ParseOrderBy.hpp)
 add_library(quickstep_parser_ParsePredicate ParsePredicate.cpp ParsePredicate.hpp)
 add_library(quickstep_parser_ParserUtil ParserUtil.cpp ParserUtil.hpp)
+add_library(quickstep_parser_ParseSample ParseSample.cpp ParseSample.hpp)
```

Comment:
The order before `quickstep_parser_ParseSample` does not follow the alphabet order. We need to fix it in another PR.